hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -43,7 +43,7 @@ class HassioAddonConfig extends LitElement {
     const valid = editor ? editor.isValid : true;
 
     return html`
-      <paper-card heading="Config">
+      <paper-card heading="${this.addon.name} Configuration">
```

Comment:
Maybe just add it as a title/heading above each page?

Instead of adding it to the title of the card.
Some add-on names might long?

This also makes it inconsistent right now with the network card. I guess we could add it there too, but that would become ugly?